CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v0.5.1] - 2021-07-06
+
+### Changed
+
+- give better error message to the user when the resource source keys are misconfigured
+
+### Fixed
+
+- remove the logic to disambiguate the error message in case the initial http request failed. `disambiguateError` function was working correctly only in special cases when user had read but not write access to the repository. Instead of trying to disambiguate the error message we list all 3 reasons that could cause this error to occur.
+
 ## [v0.5.0] - 2021-03-09
```

Comment:
We can just say:

- remove flawed logic attempting to find the real cause when receiving an opaque GitHub API error. Instead, we now print to the user all the possible reasons.